edx_proctoring/tests/test_views.py

```diff
@@ -3852,41 +3847,3 @@ def test_deletes_all_attempts(self):
         )
         attempts = ProctoredExamStudentAttempt.objects.filter(user_id=self.user.id, proctored_exam_id=self.exam_id)
         assert len(attempts) == 0
-
-    @ddt.data(
-        (ProctoredExamStudentAttemptStatus.verified, 'satisfied'),
-        (ProctoredExamStudentAttemptStatus.submitted, 'submitted'),
-        (ProctoredExamStudentAttemptStatus.declined, 'declined'),
-        (ProctoredExamStudentAttemptStatus.error, 'failed'),
-    )
-    @ddt.unpack
-    def test_credit_update(self, to_status, requirement_status):
```

Comment:
removed test here because the same logic (with more checks) is being covered in the tests I added to test_api.py